rs/sns/testing/src/utils.rs

```diff
@@ -110,6 +110,29 @@ pub async fn get_nns_neuron_hotkeys<C: CallCanisters>(
     Ok(neuron.iter().flat_map(|n| n.hot_keys.clone()).collect())
 }
 
+pub async fn get_nns_neuron_controller<C: CallCanisters>(
```

Comment:
This function could be useful for production scenarios, too (consider moving it to mervous_system_agent).

Low-prio request, feel free to skip if you run out of time